lua/astrocommunity/pack/rust/rust.lua

```diff
@@ -18,7 +18,22 @@ return {
     "simrat39/rust-tools.nvim",
     ft = { "rust" },
     init = function() utils.list_insert_unique(astronvim.lsp.skip_setup, "rust_analyzer") end,
-    opts = function() return { server = require("astronvim.utils.lsp").config "rust_analyzer" } end,
+    opts = function()
+      local extension_path = vim.fn.stdpath "data" .. "/mason/packages/codelldb/extension/"
```

Comment:
This should use the mason API for getting the package installation location rather than hard coding a path.

Reply:
Here is the mason API documentation: https://github.com/williamboman/mason.nvim/blob/main/doc/reference.md

Reply:
That should be done now